bundle.Dockerfile

```diff
@@ -1,17 +1,21 @@
 FROM scratch
 
+# Core bundle labels.
 LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
 LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=kogito-operator
 LABEL operators.operatorframework.io.bundle.channels.v1=alpha,1.x
 LABEL operators.operatorframework.io.bundle.channel.default.v1=1.x
-LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.2.0
+LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.10.0+git
```

Comment:
isn't the version 1.11.0?

Reply:
`operator-sdk-v1.10.0+git` label is for `1.11.0`. This label value is changed automatically changed with operator-sdk upgrade.